src/styles/konnectorInstall.styl

```
@@ -1,9 +1,15 @@
 @require 'tools/mixins.styl'
+@require 'settings/breakpoints.styl'
 @require 'settings/palette.styl'
 
 :local // @stylint ignore
     .col-account-connection-content
         padding 0 2rem
+
```

```suggestion
    +tiny-screen()
          padding 0
```
but, you know, make it descendant of `.col-account-connection-content`